packages/php-wasm/web/src/lib/tcp-over-fetch-websocket.spec.ts

```diff
@@ -164,23 +239,30 @@ describe('TCPOverFetchWebsocket', () => {
 		expect(responseHeaders.value).toContain('HTTP/1.1 200 OK');
 		expect(responseHeaders.done).toBe(false);
 
+		await reader.read(); // Skip the chunk length (Transfer-Encoding: chunked)
 		const responseBodyPart1 = await reader.read();
+		await reader.read(); // Skip the chunk delimiter
```

Comment:
It might be good to confirm that each of these blind reads actually skips what we think it is skipping.

Reply:
Good call! I'm not very happy with this test but I also couldn't come up with a better one on the spot.

Reply:
I think it's fine and good that the test exists. :) We can always improve things along the way.